FlymFork/src/main/res/values/strings.xml

```diff
@@ -27,31 +27,31 @@
     <string name="about_flym">About the app / Help / Contact developer. </string>
     <string name="welcome_title">Welcome to Flym Fork. Would you like to add a feed?</string>
     <string name="never">never</string>
-    <string name="update">Update</string>
+    <string name="update">update</string>
     <string name="colon">:\u00A0</string>
     <string name="menu_settings">Settings</string>
     <string name="menu_refresh">Refresh</string>
     <string name="menu_all_read">Mark all as read</string>
-    <string name="flym_feeds">Flym feeds</string>
+    <string name="flym_feeds">Flym Feeds</string>
     <string name="error">Error</string>
-    <string name="network_error">The network is not set up</string>
+    <string name="network_error">The network is not (properly) set up</string>
```

Comment:
This is not manual setup. It if the state of radio module

Reply:
Ok.